test/TestAsset/config/application.config.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+/** @psalm-suppress UnevaluatedCode */
```

Comment:
Would be better to have this in the baseline, no?

Reply:
There's currently no baseline - it actually looks like a bug to me - where is the unevaluated code???

Reply:
Yes, I saw that too, but that's why I think it should be in a baseline, and not in the file itself :D 

Suppressions are for intentional design decisions, not for avoiding some errors caused by faulty tooling.

I suggest reporting this in upstream BTW, after reproducing on the https://psalm.dev/ playground

Reply:
My initial suspicion is that any of the expressions used in here is evaluated to `: never`

Reply:
It's a weird one - I can't reproduce on psalm.dev

Moved to baseline as suggested for now 👍